bin/gobblin.sh

```diff
@@ -238,6 +238,11 @@ if [[ "$GOBBLIN_MODE_TYPE" == "$CLI" ]]; then
     fi
 fi
 
+CHECK_ENV_VARS=false
+if [ $ACTION == "start" ] || [ $ACTION == "restart" ]; then
```

Comment:
actually, only start and restart goes through the function that does this check. Also the setting of ENV variables should be optional.

Reply:
btw, there is a syntax error here as per bash 4, did you encounter this ?

Reply:
@Will-Lo , this line has syntax error as per bash 4.0, can u pls take a look.
since `CHECK_ENV_VARS` check happens only in start function anyway, do you want to remove this conditional set and set it by default to TRUE always. Also if you set the variable at  line 34, it can be overridable by `gobblin-env.sh`.